docs/java-rest/low-level/usage.asciidoc

```diff
@@ -329,6 +329,19 @@ include-tagged::{doc-tests}/RestClientDocumentation.java[rest-client-async-examp
 <2> Handle the returned exception, due to communication error or a response
 with status code that indicates an error
 
+==== Cancelling asynchronous requests
+
+The `performRequestAsync` method returns a `Cancellable` that exposes a single
+public method called `cancel`. Such method can be called to cancel the on-going
+request.
```

Comment:
Maybe mention the consequences of calling cancel: "Calling cancel causes the abort method to be called on the underlying apache httpclient request."

Reply:
Maybe add a comment explaining what cancellation means server side, for search requests will be aborted but for other APIs, closing the channel won't prevent the request to continue running on the server.